cross/arm-softfp/toolchain.cmake

```diff
@@ -0,0 +1,31 @@
+set(CROSS_ROOTFS $ENV{ROOTFS_DIR})
+
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_VERSION 1)
+set(CMAKE_SYSTEM_PROCESSOR armv7l)
+
+## Specify the toolchain
+set(TOOLCHAIN "arm-linux-gnueabi")
+
+add_compile_options(-target ${TOOLCHAIN})
+add_compile_options(-mthumb)
+add_compile_options(-mfpu=vfpv3)
+add_compile_options(--sysroot=${CROSS_ROOTFS})
+
+set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -target ${TOOLCHAIN}")
+set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -B${CROSS_ROOTFS}/usr/lib/${TOOLCHAIN}")
```

Comment:
@ianhays True. That path should have been `/usr/lib/gcc/${TOOLCHAIN}` instead of `/usr/lib/${TOOLCHAIN}`. I have made this correction in the latest version. Thank you :)